examples/images_contours_and_fields/contour_image.py

```diff
@@ -34,13 +34,11 @@
 norm = cm.colors.Normalize(vmax=abs(Z).max(), vmin=-abs(Z).max())
 cmap = cm.PRGn
 
-fig = plt.figure()
+fig, ((ax1, ax2), (ax3, ax4)) = plt.subplots(nrows=2, ncols=2)
```

Comment:
I'm not a huge fan of this syntax, versus just `fig, axs = ` and then setting `ax = axs[row, col]`.  But not a show-stopper at all.

Reply:
I'm with you on that. We do it both ways. And while I think consistency is good, maybe showing options are important too

Reply:
I'm not huge fan of putting arrays in variable names (`ax1, ax2, ax3, ax4`) rather than just the type (`ax[0]`). The former is so much harder to change later on, so I'm not sure we should actually be promoting it. If the row/column nature is a problem, you can always do `ax = ax.flatten()`. My $0.02.